src/locale/messages.fr.xlf

```diff
@@ -1,313 +1,365 @@
         <note priority="1" from="description">tooltip critical</note>
       </trans-unit>
       <trans-unit datatype="html" id="fff721af9d16ca1cd0983fce54d10c5e857ba341">
         <source>Error - means a problem that is very likely (or possibly certain) to negatively affect the function of the zone being tested, but not so severe that the entire zone becomes unresolvable</source>
-        <target state="new">Error - une erreur qui a de grande change d&apos;affecter la zone testée, mais pas suffisamment pour la rendre inutilisable</target>
+        <target state="translated">Erreur - une erreur qui a de grande change d'affecter la zone testée, mais pas suffisamment pour la rendre inutilisable</target>
         <note priority="1" from="description">tooltip error</note>
       </trans-unit>
       <trans-unit datatype="html" id="281f2af4328daa895401b765738b1eb9cc55f489">
-        <source>Info - is something that may be of interest to the zone&apos;s administrator but that definitely does not indicate a problem.</source>
-        <target state="new">Info - contient des informations qui peuvent être d&apos;intérêt pour la personne lançant le test, ces messages ne sont pas des erreurs</target>
+        <source>Info - is something that may be of interest to the zone's administrator but that definitely does not indicate a problem.</source>
+        <target state="translated">Info - contient des informations qui peuvent être d'intérêt pour la personne lançant le test, ces messages ne sont pas des erreurs.</target>
         <note priority="1" from="description">tooltip info</note>
       </trans-unit>
       <trans-unit datatype="html" id="5f27bfb99492b3cb2b7ae776e56665fa23c93c48">
-        <source>Notice - means something that should be known by the zone&apos;s administrator but that need not necessarily be a problem at all</source>
-        <target state="new">Notice - destiné à informer la personne en charge de la zone, ne correspond pas nécessairement à un problème de configuration</target>
+        <source>Notice - means something that should be known by the zone's administrator but that need not necessarily be a problem at all</source>
+        <target state="translated">Note - destiné à informer la personne en charge de la zone, ne correspond pas nécessairement à un problème de configuration</target>
         <note priority="1" from="description">tooltip notice</note>
       </trans-unit>
       <trans-unit datatype="html" id="5f8e2137b25e61faf62e17cea5f550a5ae2fbf9c">
         <source>Warning - means something that will under some circumstances be a problem, but that is unlikely to be noticed by a casual user.</source>
-        <target state="new">Warning - destiné aux erreurs détectées dans certains cas d&apos;usage spécifiques</target>
+        <target state="translated">Avertissement - destiné aux erreurs détectées dans certains cas d'usage spécifiques.</target>
```

Comment:
Remove trailing dot

Reply:
Disagreeing. However the translation could be improved. How about: “Avertissement – un élément pouvant causer un dysfonctionnement dans certaines circonstances, mais qu’un utilisateur non averti a peu de chances de repérer.”

Reply:
> However the translation could be improved.

Done